init_test.go

```diff
@@ -0,0 +1,15 @@
+package composer_test
+
+import (
+	"testing"
+
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+)
+
+func TestUnitcomposer(t *testing.T) {
+	suite := spec.New("composer", spec.Report(report.Terminal{}))
+	suite("Detect", testDetect)
```

Comment:
May want to run these in parallel since you're setting env. vars

Reply:
```suggestion
	suite("Detect", testDetect, spec.Sequential())
```

Reply:
To clarify - in parallel or in sequence? Sequential would make sense if the env is dirty.

Reply:
omg I meant sequential. So confusing, sorry!!!